src/UnderlineNav2/LoadingCounter.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import styled from 'styled-components'
+
+import {useTheme, Theme} from '../ThemeProvider'
+
+export const LoadingCounter = () => {
+  const {theme} = useTheme()
+
+  const getLoaderStyle = (currentTheme?: Theme) => {
+    return `@keyframes loading {
+      from {
+        opacity: 0.4;
+      }
+      to {
+        opacity: 0.8;
+      }
+    }
+    animation: loading 1.2s linear infinite alternate;
+    background-color: ${currentTheme?.colors.neutral.emphasis};
+    border-color: ${currentTheme?.colors.border.default};
+    width: 22px;
+    height: 16px;
+    display: inline-block;
+    border-radius: 20px;`
```

Comment:
Is 20 an arbitrary number?

Reply:
No, I got it from the [CounterLabel's styles](https://github.com/primer/react/blob/main/src/CounterLabel.tsx#L38) I currently don't  have much design details for the loading counters except a demo on [the Figma file](https://www.figma.com/file/N9JSJxBe6XAIeUjkvl4Bsy/UnderlineNav-Updates-Q1?node-id=8%3A12282) So I tried to use CounterLabel's style for a minimal layout shift. I am not too sure about the animation colours either but I'll check it when @danielguillan is back